### PR TITLE
Issue 2 view their entries

### PR DESCRIPTION
**Pull request by owen-adams2021**
image
image
image
image

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -58,10 +83,28 @@ textarea {
...
+  padding: 0.5rem 1rem;
+  background-color: #b0b7bc;
+  color: #ce0f3d;
+  text-transform: uppercase;
+  letter-spacing: 0.15rem;
+  border: 0 solid #b0b7bc;
+  border-radius: 0.175rem;
+  box-shadow: 2px 2px 2px #ce0f3d;
+  cursor: pointer;
+  position: absolute;
```
Comment (codingcodymiller) [inline css/styles.css:98]: This shouldn't need to be `absolute`

File: js/main.js [hunk 2]
```diff
@@ -26,3 +32,57 @@ $submit.addEventListener('submit', function (event) {
   $imageOutput.setAttribute('src', 'images/placeholder-image-square.jpg');
   $submit.reset();
```
Comment (codingcodymiller) [inline js/main.js:33]: You will need to crate the DOM tree for the new entry here. As it is right now, your new entry will only be visible when the page is refreshed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

File: js/main.js (hunk 2 above)
Comment (codingcodymiller) [reply js/main.js:33]: *create

File: css/styles.css (hunk 1 above)
Comment (owen-adams2021) [reply css/styles.css]: Removed `position: absolute;` and added `justify-content: space-between;' to the targeted CSS Selector.

File: js/main.js (hunk 2 above)
Comment (owen-adams2021) [reply js/main.js:34]: Created `$ul.prepend(addEntry(formObj));` in `$submit.addEventListener` 's function to load newly submitted entry immediately to Entries page.

**Comment by owen-adams2021** [comment]
Screenshots of new functionality.

image
image

**Review by codingcodymiller — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**